dsl/src/test/java/org/mule/test/config/ast/ArtifactAstValidationsTestCase.java

```diff
@@ -53,6 +84,65 @@
 @Story(DSL_VALIDATION_STORY)
 public class ArtifactAstValidationsTestCase extends AbstractMuleContextTestCase {
 
+  private Injector injector;
+
+  @Rule
+  public SystemProperty systemProperty = new SystemProperty(ENFORCE_EXPRESSION_VALIDATION_PROPERTY, "true");
```

Comment:
why is this needed? isn't this the default?

Reply:
Actually this can be removed as we are injecting FeatureFlaggingService